### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by amolw-cuelogic**
@nikhilb-cuelogic  please review. the code.

### CODE REVIEW

File: Source/Server/Cuelogic.Clrm.Api.Tests/Controllers/MasterGroupControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Cuelogic.Clrm.Api.Controllers;
+using Cuelogic.Clrm.Service.Service;
```
Comment (nikhilb-cuelogic) [inline Source/Server/Cuelogic.Clrm.Api.Tests/Controllers/MasterGroupControllerTest.cs]: Why service contain service

File: Source/Server/Cuelogic.Clrm.Api/Controllers/MasterGroupController.cs [hunk 2]
```diff
@@ -20,6 +20,7 @@ public class MasterGroupController : ApiBaseController
         IMasterGroup MasterObj;
         public MasterGroupController(IMasterGroup iobjMasterGroup)
```
Comment (nikhilb-cuelogic) [inline Source/Server/Cuelogic.Clrm.Api/Controllers/MasterGroupController.cs]: Use readable naming convention

File: Source/Server/Cuelogic.Clrm.Api.Tests/Controllers/MasterGroupControllerTest.cs (hunk 1 above)
Comment (amolw-cuelogic) [reply Source/Server/Cuelogic.Clrm.Api.Tests/Controllers/MasterGroupControllerTest.cs]: @nikhilb-cuelogic  - I have changes the folder hierarchy as per our discussion which accordingly changes the namespace

File: Source/Server/Cuelogic.Clrm.Api/Controllers/MasterGroupController.cs (hunk 2 above)
Comment (amolw-cuelogic) [reply Source/Server/Cuelogic.Clrm.Api/Controllers/MasterGroupController.cs]: Changed the variable names

**Pull request merged**